coder_sniffer/DrupalPractice/Test/Objects/StrictSchemaDisabledUnitTest.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Test class with strict config schema checking disabled.
+ */
+class TestFalse {
+
+    /**
+     * Disable strict config checking during testing.
+     */
+    protected $strictConfigSchema = FALSE;
+}
+
+class TestNull {
+    protected $strictConfigSchema = NULL;
+
+    public function strictConfigSchema()
+    {
```

Comment:
Sorry, missed this in my first review: can you use Drupal coding standards in the test files? 2 space indentation, braces on the line before etc.

If you create StrictSchemaDisabledUnitTest.inc.fixed then you see what it should look like :-)